UnityGLTF/Assets/UnityGLTF/Scripts/GLTFSceneImporter.cs

```diff
@@ -180,9 +180,13 @@ public async Task LoadSceneAsync(int sceneIndex = -1, bool showSceneObj = true,
 				{
 					await LoadJson(_gltfFileName);
 				}
-				await _LoadScene(sceneIndex, showSceneObj);
 
-				Cleanup();
+				if (_assetCache == null)
+				{
+					_assetCache = new AssetCache(_gltfRoot);
```

Comment:
did you ensure with this change the asset cache still gets cleaned up properly?

Reply:
Yes and no. The asset cache persists indefinitely during the lifetime of the importer, then is disposed when the importer is disposed. Since you can't load a second glTF document from a single importer anyway, it works out.